### ISSUE TITLE
radio: no sound

### ISSUE THREAD

**Issue by Solarunit** [Taiko2k/TauonMusicBox#996]
Tauon v7.6.5 flatpak

No sound from any radio. Can't remember if it worked yesterday.
In the terminal:

```
Start radio
https://kathy.torontocast.com:2005/stream
Websocket closed
pa: FFMPEG has finished
Exception in thread Thread-23 (send_heartbeat):
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/app/bin/t_modules/t_main.py", line 35196, in send_heartbeat
    ws.send("{\"op\":9}")
  File "/app/lib/python3.10/site-packages/websocket/_app.py", line 204, in send
    raise WebSocketConnectionClosedException(
websocket._exceptions.WebSocketConnectionClosedException: Connection is already closed.

```

**Comment by Taiko2k** [comment]
Ah, I unintentionally left some debug code enabled. Due to reasons, this bug only surfaces in the Flatpak version so thats why I missed the issue in my testing.

Ive now released v7.6.6 with a hotfix for this.

**Comment by Solarunit** [comment]
Thanks for the quick update!
